### PR TITLE
Issue 59 feat/UI post collection

### PR DESCRIPTION
**Pull request by KonTra7cf**
Waiting for useLatestPosts hook so for now I am passing empty array to PostCollection in FeedScreen

### CODE REVIEW

**Review by Veryyapeee — APPROVED** [review]

**Review by PoProstuMieciek — APPROVED** [review]

File: src/components/organisms/PostCollection/PostCollection.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+import { Post } from '../..';
+
+export const PostCollection = ({ userPosts }) => {
+    return userPosts.map(({ postID }) => <Post postID={postID} />);
```
Comment (PoProstuMieciek) [inline src/components/organisms/PostCollection/PostCollection.jsx:5]: ~~`{ postID }`~~

`postID`

**Review by PoProstuMieciek — CHANGES_REQUESTED** [review]

File: src/components/organisms/PostCollection/PostCollection.jsx (hunk 1 above)
Comment (PoProstuMieciek) [reply src/components/organisms/PostCollection/PostCollection.jsx:5]: line 5

Comment (PoProstuMieciek) [reply src/components/organisms/PostCollection/PostCollection.jsx:5]: we pass an array of strings, not an array of { postID: string } objects

Comment (PoProstuMieciek) [reply src/components/organisms/PostCollection/PostCollection.jsx:5]: small typo, everything seems fine, thanks!

Comment (KonTra7cf) [reply src/components/organisms/PostCollection/PostCollection.jsx]: Done

**Review by PoProstuMieciek — APPROVED** [review]

**Pull request merged**